publications-2021.bib

```diff
@@ -877,6 +877,180 @@ @article{heltai2021vascualar
+  year = {2021},
+  month = may,
+  publisher = {American Geophysical Union ({AGU})},
+  volume = {126},
+  number = {5},
+  author = {Matthew J. Comeau and Claudia Stein and Michael Becken and Ulrich Hansen},
+  title = {Geodynamic Modeling of Lithospheric Removal and Surface Deformation: Application to Intraplate Uplift in {C}entral {M}ongolia},
+  journal = {Journal of Geophysical Research: Solid Earth}
+}
+
+@article{STEIN2022229276,
+title = {Numerical study on the style of delamination},
+journal = {Tectonophysics},
+pages = {229276},
+year = {2022},
+issn = {0040-1951},
+doi = {10.1016/j.tecto.2022.229276},
+url = {https://www.sciencedirect.com/science/article/pii/S0040195122000701},
+author = {Claudia Stein and Matthew Comeau and Michael Becken and Ulrich Hansen},
+keywords = {Numerical modelling delamination, Rayleigh-Taylor instability phase transition rheology}
+}
+
+@article{https://doi.org/10.1029/2021JB023244,
+author = {Zha, Caicai and Lin, Jian and Zhou, Zhiyuan and Xu, Min and Zhang, Xubo},
+title = {Effects of Hotspot-Induced Long-Wavelength Mantle Melting Variations on Magmatic Segmentation at the {R}eykjanes Ridge: Insights From 3D Geodynamic Modeling},
+journal = {Journal of Geophysical Research: Solid Earth},
+volume = {127},
+number = {3},
+pages = {e2021JB023244},
+doi = {10.1029/2021JB023244},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2021JB023244},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2021JB023244},
+note = {e2021JB023244 2021JB023244},
+year = {2022}
+}
+
+@article{https://doi.org/10.1029/2022GL098003,
+author = {Heyn, Bj{\"o}rn H. and Conrad, Clinton P.},
+title = {On the relation between basal erosion of the lithosphere and surface heat flux for continental plume tracks},
+journal = {Geophysical Research Letters},
+volume = {49},
+number = {7},
+year = {2022},
+pages = {e2022GL098003},
+keywords = {plume-lithosphere interactions, mantle plumes, geothermal heat flux, lithospheric thinning, continental plume tracks, upper mantle viscosity structure},
+doi = {10.1029/2022GL098003},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2022GL098003},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2022GL098003},
+note = {e2022GL098003 2022GL098003}
+}
+
+@article{NEGREDO2022117506,
+title = {On the origin of the {C}anary {I}slands: {I}nsights from mantle convection modelling},
+journal = {Earth and Planetary Science Letters},
+volume = {584},
+pages = {117506},
+year = {2022},
+issn = {0012-821X},
+doi = {10.1016/j.epsl.2022.117506},
+url = {https://www.sciencedirect.com/science/article/pii/S0012821X2200142X},
+author = {Ana M. Negredo and Jeroen {van Hunen} and Juan Rodríguez-González and Javier Fullea},
+keywords = {Canary Islands, edge-driven convection, hot spots, mantle convection}
+}
+
+@article{Behr_2022,
+  doi = {10.1093/gji/ggac075},
+  url = {https://doi.org/10.1093/gji/ggac075},
+  year = 2022,
+  month = {feb},
```

Comment:
```suggestion
  month = feb,
```
From my experience, bibtex doesn't like when you provide the month information in brackets or in apostrophes. With abbreviations or integers like this, it is capable to identify the right month, otherwise it takes the entry as _set_ and will just display the text. I think biblatex is more lenient about this, but bibtex is usually quite strict.

Reply:
Really? I thought quotes or braces are optional for single values but there shouldn't be any difference otherwise.

Reply:
I found a paragraph in the bibtex FAQ on CTAN https://www.ctan.org/pkg/bibtex
> Q13: Should I use words or numerals for the month, edition, etc., fields, and why?
You should always use the three letter month macros (undelimited) for the name of the month (jan, feb,
mar, apr, may, jun, jul, aug, sep, oct, nov, dec) as this lets each .bst file format the month names as is
required for that bibliography style. [...]

Reply:
Huh, interesting. Thank you for looking it up.